### PR TITLE
Clarify CCoinsViewMemPool documentation.

### PR DESCRIPTION
**Pull request by TheBlueMatt**
Thanks to @sdaftuar for correcting my misunderstanding.

### CODE REVIEW

**Comment by sipa** [comment]
ACK

**Comment by morcos** [comment]
ACK

**Review by gmaxwell — APPROVED** [review]
ACK

**Comment by sdaftuar** [comment]
ACK

**Comment by fanquake** [comment]
utACK 381b8fc

File: src/rpc/blockchain.cpp [hunk 1]
```diff
@@ -991,7 +991,7 @@ UniValue gettxout(const JSONRPCRequest& request)
     if (fMempool) {
         LOCK(mempool.cs);
         CCoinsViewMemPool view(pcoinsTip, mempool);
-        if (!view.GetCoin(out, coin) || mempool.isSpent(out)) { // TODO: filtering spent coins should be done by the CCoinsViewMemPool
+        if (!view.GetCoin(out, coin) || mempool.isSpent(out)) {
```
Comment (dougstrong77) [inline src/rpc/blockchain.cpp]: Fuck

**Pull request merged**